test/typedarray/rlexe.xml

```diff
@@ -438,6 +438,7 @@ Below test fails with difference in space. Investigate the cause and re-enable t
   <test>
     <default>
       <files>sort.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
```

Comment:
Not sure if you picked up what this does or just copied it from another test.

Passing `-args item_1 item_2 item_3 -endargs` to `ch.xex` on the command line populates an array `WScript.Arguments` that's accessible within javascript with item_1, item_2 and item_3 - this case it's just putting `summary` as the 0 element.

This is used here as a switch - running testrunner tests without the switch will print out details per test to the command line whereas with the switch it just prints "pass" (or details of a failure) - the detailed printout is useful when running the test individually whereas we need the "pass" only output for the CI harness.

Reply:
I also copied that from `Array\array_sort.js`. With the full output, the test was claimed to have failed.

Reply:
Thanks for the clarification!
It took me quite a while to understand why the test was supposedly failed....

Reply:
The tests are annoying, but very helpful.
My other PR would have introduced another assertion error otherwise!